tna/collections/management/commands/fetch_results_pages.py

```diff
@@ -67,29 +69,45 @@ def fetch_page_data(url):
 class Command(BaseCommand):
     def handle(self, *args, **kwargs):
         login()
+        num_urls_created = 0
+        num_urls_updated = 0
+        num_urls_errored = 0
+        num_urls_fetched = 0
 
         home_page = HomePage.objects.get()
 
         results_index_page = ResultsIndexPage.objects.first()
         if not results_index_page:
-            results_index_page = ResultsIndexPage(title="Insights")
+            results_index_page = ResultsIndexPage(title="Highlights")
             home_page.add_child(instance=results_index_page)
 
         for url in fetch_urls():
             print(f"Processing {url}")
-
-            page_data = fetch_page_data(url)
-
+            num_urls_fetched += 1
             try:
-                results_page = ResultsPage.objects.get(source_url=url)
-            except ResultsPage.DoesNotExist:
-                results_page = ResultsPage(source_url=url)
-
-            results_page.slug = page_data["slug"]
-            results_page.title = page_data["title"]
-            results_page.body = page_data["body"]
-
-            if not results_page.id:
-                results_index_page.add_child(instance=results_page)
-            else:
-                results_page.save()
+
+                page_data = fetch_page_data(url)
+
+                try:
+                    results_page = ResultsPage.objects.get(source_url=url)
+                except ResultsPage.DoesNotExist:
+                    results_page = ResultsPage(source_url=url)
+
+                results_page.slug = page_data["slug"]
+                results_page.title = page_data["title"]
+                results_page.body = page_data["body"]
+
+                if not results_page.id:
+                    results_index_page.add_child(instance=results_page)
+                    num_urls_created += 1
```

Comment:
Throwaway comment: This whole thing could be so much cleaner if we weren't using `Page` models here :(